CHANGES.md

```diff
@@ -475,7 +551,7 @@ Developer-oriented changes: Testing System
 version 1.20.0
 ==============
 
-Twenty-third public release of Chapel, September 19, 2019
+Twenty-third public release of Chapel, September 19, 2019  
```

Comment:
Was the additional space intentional?

Reply:
Yeah, it forces a linefeed in the rendered `.md` file.  Wish I'd known about this years ago...!

(mostly because I know it's going to drive @ben-albrecht crazy!)

Reply:
All those bytes going to waste!!!